assembly/std/index.ts

```diff
@@ -114,15 +133,14 @@ export function functionExists(address: Address, func: string): bool {
 /**
  * Creates a new smart contract.
  *
- * Take a base64 string representing the module binary and create an entry in
- * the ledger.
+ * Takes a byte array which is the bytecode of the contract to create.
  *
  * The context allow you to write in this smart contract while you're executing
  * the current bytecode.
  *
- * @param bytecode -
+ * @param bytecode - The byte code of the contract to create.
  *
- * @returns Smart contract address
+ * @returns the newly ledger-registered contract address
```

Comment:
I would change to => the address of the newly created smart contract on the ledger

Reply:
Done!